servers/Azure.Mcp.Server/TROUBLESHOOTING.md

```diff
@@ -881,10 +881,25 @@ To export telemetry to Azure Monitor, set the `APPLICATIONINSIGHTS_CONNECTION_ST
 
 ### Development in VS Code
 
+#### Running Azure MCP Server Locally for Development
+
+When developing or debugging the Azure MCP Server locally, you can use `dotnet run` which automatically uses the configuration in `Properties/launchSettings.json`:
+
+```bash
+dotnet run --project servers/Azure.Mcp.Server/src/
+```
+
+The `launchSettings.json` file automatically configures:
+- **Command line arguments:** `server start --run-as-remote-http-service --outgoing-auth-strategy UseHostingEnvironmentIdentity`
+- **Environment variables** for Azure AD authentication and ASP.NET Core settings
+- **HTTP endpoint:** `http://localhost:1031` for easier debugging and testing
+
+This is the recommended approach for local development as it sets up the server in HTTP mode with all necessary authentication and configuration automatically.
```

Comment:
Potentially add a sample config for VS Code to connect to the HTTP MCP server or reference the section in the `CONTRIBUTING.md` for easier discovery.